ci/authn-k8s/secrets.yml

```
@@ -1,4 +1,16 @@
-GCLOUD_CLUSTER_NAME: !var ci/google-container-engine-testbed/gcloud-cluster-name
-GCLOUD_ZONE: !var ci/google-container-engine-testbed/gcloud-zone
-GCLOUD_PROJECT_NAME: !var ci/google-container-engine-testbed/gcloud-project-name
-GCLOUD_SERVICE_KEY: !var:file ci/google-container-engine-testbed/gcloud-service-key
+#GCLOUD_CLUSTER_NAME: !var ci/google-container-engine-testbed/gcloud-cluster-name
+#GCLOUD_ZONE: !var ci/google-container-engine-testbed/gcloud-zone
+#GCLOUD_PROJECT_NAME: !var ci/google-container-engine-testbed/gcloud-project-name
+#GCLOUD_SERVICE_KEY: !var:file ci/google-container-engine-testbed/gcloud-service-key
+
+openshift43:
+  K8S_VERSION: '1.16'
+  OPENSHIFT_CLI_URL: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.3/openshift-client-linux.tar.gz
+  OPENSHIFT_URL: !var ci/openshift/4.3/api-url
+  OPENSHIFT_USERNAME: !var ci/openshift/4.3/username
+  OPENSHIFT_PASSWORD: !var ci/openshift/4.3/password
+  OPENSHIFT_REGISTRY_URL: !var ci/openshift/4.3/registry-url
```

Edit, add lines to run tests with OC version X
```suggestion
openshiftX:
  K8S_VERSION: '1.16'
  OPENSHIFT_CLI_URL: https://mirror.openshift.com/pub/openshift-vX/clients/ocp/latest-X/openshift-client-linux.tar.gz
  OPENSHIFT_URL: !var ci/openshift/X/api-url
  OPENSHIFT_USERNAME: !var ci/openshift/X/username
  OPENSHIFT_PASSWORD: !var ci/openshift/X/password
  OPENSHIFT_REGISTRY_URL: !var ci/openshift/X/registry-url
```